### PR TITLE
[ora] introduce typings for v1

### PR DESCRIPTION
**Pull request by BendingBender**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/sindresorhus/ora
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by BendingBender** [comment]
_types/ora/index.d.ts_

to authors (@screendriver). Could you review this PR?
:+1: or :-1:?

**Review by screendriver — CHANGES_REQUESTED** [review]
Please fix [indentation](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes) (4 spaces)

**Comment by BendingBender** [comment]
@screendriver Do you mind if I change the URL to your profile in the header to you github profile so that dt-bot can find and notify you automatically the next time?

File: types/ora/index.d.ts [hunk 1]
```diff
@@ -1,34 +1,133 @@
...
-// Definitions by: Basarat Ali Syed <https://github.com/basarat/>, Christian Rackerseder <https://www.echooff.de/>
+// Definitions by: Basarat Ali Syed <https://github.com/basarat/>
+//                 Christian Rackerseder <https://www.echooff.de/>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
+type SpinnerName =
+  'dots'
```
Comment (screendriver) [inline types/ora/index.d.ts]: 2 spaces

**Comment by screendriver** [comment]
> [@screendriver](https://github.com/screendriver) Do you mind if I change the URL to your profile in the header to you github profile so that dt-bot can find and notify you automatically the next time?

Of course, thank you ;-) I thought I can add any URL but it seems that it must be my [profile URL](https://github.com/screendriver) here at GitHub.

There are still 2 spaces at `type SpinnerName`. The rest of the changes looks good to me 👍

**Pull request merged**